### PR TITLE
Patient plan redesign and modularization

### PR DESCRIPTION
**Pull request by jakepeyser**

### CODE REVIEW

File: app/components/plan/Workout.js [hunk 1]
```diff
@@ -1,52 +1,80 @@
...
-          <Link to={`/plan/treatments/${treatment.id}/workout`}>
-            <StepsActionButton>
-              <FontIcon className={'material-icons'}>schedule</FontIcon>
-            </StepsActionButton>
-          </Link>
-      }
+      <div className="workout-index">#{num}</div>
+      <div className="workout-pic-wrapper">
+        <img
+          className="workout-pic"
```
Comment (mkewls) [inline app/components/plan/Workout.js]: would suggest also adding 'img-responsive' to className here to help control img height/width relative to its container sizing.

File: app/components/widgets/InfoItem.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+export default ({ icon, iconColor, label, content }) => (
+  <div className="info-item">
+  {
+    icon ?
+      <FontIcon className="material-icons"
+        color={iconColor ? iconColor : 'black'}
+        style={{marginRight: '5px', fontSize: '20px'}}>{ icon }</FontIcon> : null
+  }
+    <p><span>{`${label}:`}&nbsp;</span>{ content }</p>
+  </div>
```
Comment (EmilyDev) [inline app/components/widgets/InfoItem.js]: why do we make this a new file instead addint it to material-style?

Comment (jakepeyser) [inline app/components/widgets/InfoItem.js]: because this isn't wrapping a material element

File: app/components/plan/Workout.js (hunk 1 above)
Comment (jakepeyser) [inline app/components/plan/Workout.js]: Just tried using `img-responsive` here and it doesn't actually make a difference bc Im already adjusting min/max height. I agree something does need to be done about the various image sizes here, but we can figure that out later this week

**Pull request merged**